application/templates/pdb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.pdb.enabled }}
+{{- if or .Values.pdb.minAvailable .Values.pdb.maxUnavailable }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: '{{ template "application.name" . }}-pdb'
+spec:
+  {{- if .Values.pdb.minAvailable }}
+  minAvailable: {{ .Values.pdb.minAvailable }}
+  {{- else }}
```

Comment:
can't min and max specified together?

Reply:
no we cannot : https://kubernetes.io/docs/tasks/run-application/configure-pdb/